input/pagecontent/terminology.md

```diff
@@ -7,93 +7,131 @@ As well as these included definitions this specification typically references [N
lthterminologies.gov.au/fhir/ValueSet/clinical-finding-1), [Clinical Specialty](https://healthterminologies.gov.au/fhir/ValueSet/clinical-specialty-1), [Contact Relationship Type](https://healthterminologies.gov.au/fhir/ValueSet/contact-relationship-type-3), [Encounter Type](https://healthterminologies.gov.au/fhir/ValueSet/encounter-type-1), [Evaluation Procedure](ValueSet/evaluation-procedure-1), [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1), [Health Program Participation Status](https://healthterminologies.gov.au/fhir/ValueSet/health-program-participation-status-1), [Healthcare Organisation Role Type](ValueSet/healthcare-organisation-role-type-1), [Imagine Procedure](https://healthterminologies.gov.au/fhir/ValueSet/imaging-procedure-1), [Imaging Anatomic Region Of Interest](https://healthterminologies.gov.au/fhir/ValueSet/imaging-anatomic-region-of-interest-1), [Immunisation Anatomical Site](https://healthterminologies.gov.au/fhir/ValueSet/immunisation-anatomical-site-1), [Immunisation Route of Administration](https://healthterminologies.gov.au/fhir/ValueSet/immunisation-route-of-administration-1), [Indicator of Hypersensitivity or Intolerance to Substance](https://healthterminologies.gov.au/fhir/ValueSet/indicator-hypersensitivity-intolerance-to-substance-2), [Medication Form](https://healthterminologies.gov.au/fhir/ValueSet/medication-form-1), [Medication Reason Taken](https://healthterminologies.gov.au/fhir/ValueSet/medication-reason-taken-1), [Medicine Substitution Reason](https://healthterminologies.gov.au/fhir/ValueSet/medicine-substitution-reason-1), [Participation in Health Program](https://healthterminologies.gov.au/fhir/ValueSet/participation-in-health-program-1), [Procedure](https://healthterminologies.gov.au/fhir/ValueSet/procedure-1), [Reason for Non Participation in Health Program](https://healthterminologies.gov.au/fhir/ValueSet/reason-non-participation-health-program-1), [Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1), [Reason Vaccine Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-administered-1), [Reason Vaccine Not Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-not-administered-3), [Related Person Relationship Type](https://healthterminologies.gov.au/fhir/ValueSet/related-person-relationship-type-1), [Service Type](https://healthterminologies.gov.au/fhir/ValueSet/service-type-1), [Specimen Collection Procedure](https://healthterminologies.gov.au/fhir/ValueSet/specimen-collection-procedure-1), [Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1), [Specimen Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-type-1), [Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1), [Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)|NCTS, FHIR|
```

Comment:
Remove the term "Australian extension" that is not part of the code system name.

Reply:
Amended, thanks.